### PR TITLE
Add initial support for upcoming Charge SOM platform

### PR DESCRIPTION
**Pull request by mhei**
This PR mentions the upcoming Charge SOM platform in the documentation and adds the required Yocto layer for TPM support.

While at, two small maintainance fixes are included.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,24 +21,25 @@ A.1 [How to change kernel configurations](#kernel)
...
+As the Yocto Project is based on the concept of [layers](https://docs.yoctoproject.org/dev-manual/common-tasks.html#understanding-and-creating-layers), the following table lists all the layers used to create a basic distribution based on chargebyte’s Linux platforms.
 
 | Layer | Description | Repository |
 |--|--|--|
-| meta-chargebyte | BSP layer for Tarragon & EVAcharge SE | https://github.com/chargebyte/meta-chargebyte |
-| meta-chargebyte-distro | Distribution adaptations layer | https://github.com/chargebyte/meta-chargebyte-distro |
+| poky | Build tool and metadata included in a reference distribution | https://git.yoctoproject.org/poky |
 | meta-freescale | Layer containing NXP hardware support metadata | https://git.yoctoproject.org/cgit/cgit.cgi/meta-freescale |
 | meta-openembedded | Collection of layers to supplement OE-Core with additional packages | https://github.com/openembedded/meta-openembedded |
+| meta-security | Collection of security related layers, here used for e.g. TPM support | https://git.yoctoproject.org/meta-security |
```
Comment (mooraby) [inline README.md]: I would say `Collection of security-related layers e.g. for TMP support`

File: README.md [hunk 2]
```diff
@@ -21,24 +21,25 @@ A.1 [How to change kernel configurations](#kernel)
...
 | meta-rauc| Layer controlling and performing secure software updates for embedded Linux | https://github.com/rauc/meta-rauc |
-| poky | Build tool and metadata included in a reference distribution | https://git.yoctoproject.org/poky |
+| meta-chargebyte | BSP layer for EVAcharge SE, Tarragon & Charge SOM | https://github.com/chargebyte/meta-chargebyte |
+| meta-chargebyte-distro | Distribution adaptations layer | https://github.com/chargebyte/meta-chargebyte-distro |
 
 This layering approach increases flexibility to expand your project. You can add layers, which in turn would add packages essential for the distribution you want to build. Layers are usually available as repositories. Information on how to include or remove layers will be given in [Section 3.3](#addorremove). Note that no charging capabilities will be included in the Linux distribution created by this setup.
```
Comment (mooraby) [inline README.md:44]: I think this needs to be adapted, right? If Charge SOM will be solely flashed with EVerest, then we do have charging capabilities. This point makes me question if this branch should have been based on `kirkstone-everest` and not `kirkstone`.

File: README.md [hunk 3]
```diff
@@ -159,7 +164,9 @@ The resulting image will be found in `yocto/build/tmp/deploy/image/<machine>`, h
 
 ### Flashing an Image <a name="flash"></a>
 
-The internal storage of the products Charge Control C (Tarragon) and M (EVAcharge SE) is divided into several partitions and uses RAUC to handle updates. If you are interested in adding RAUC to your own image you can take a look at RAUC's documentation https://rauc.readthedocs.io/en/latest/integration.html#yocto .To flash an image it is important to identify the currently not in use partition because this will be our target partition. The following tables describe the partitioning for each Charge Control M and Charge Control C:
+The internal storage of the products Charge Control C (Tarragon) and M (EVAcharge SE) is divided into several partitions and uses RAUC to handle updates.
```
Comment (mooraby) [inline README.md]: I think this whole paragraph can be also extended to include info about Charge SOM.

**Review by mooraby — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -182,6 +187,18 @@ The internal storage of the products Charge Control C (Tarragon) and M (EVAcharg
 | /dev/mmcblk0p6 | 128 MB | Logging file system A (/var/log) |
 | /dev/mmcblk0p7 | 128 MB | Logging file system B (/var/log) |
 
+A similar partitioning is used on the Charge SOM based products:
```
Comment (lategoodbye) [inline README.md]: I think we should mention that this partitioning only apply to the eMMC.

File: README.md (hunk 2 above)
Comment (mhei) [reply README.md:44]: No, I still want to keep the current approach of having a BSP-only layer without any charging stack or whatever. When this branch is merged, then we merge it into kirkstone-everest. At this point, the adaptions for the readme in this branch are made.

File: README.md (hunk 3 above)
Comment (mhei) [reply README.md]: I slightly changed the paragraph

File: README.md (hunk 4 above)
Comment (mhei) [reply README.md]: I now mentioned the eMMC

**Review by lategoodbye — APPROVED** [review]

**Review by mooraby — APPROVED** [review]

**Pull request merged**